hack/docgen/gen.go

```diff
@@ -17,16 +17,64 @@ limitations under the License.
 package main
 
 import (
+	"io/fs"
+	"io/ioutil"
 	"log"
+	"path/filepath"
+	"strings"
 
 	"github.com/oam-dev/kubevela/references/cli"
 
 	"github.com/spf13/cobra/doc"
 )
 
 func main() {
+	rootPath := "../kubevela.io/docs/cli/"
```

Comment:
read root path from args and default is "../kubevela.io/docs/cli/"

Reply:
sure